package.json

```diff
@@ -1,21 +1,42 @@
 {
-  "name": "jquery-plugin",
-  "version": "0.0.0-ignored",
+  "name": "jquery.addressfield",
+  "version": "0.0.0",
+  "homepage": "https://tableau-mkt.github.io/jquery.addressfield",
   "engines": {
     "node": ">= 0.8.0"
   },
   "scripts": {
     "test": "grunt test"
   },
+  "description": "The simple, configurable, dynamic postal address form plugin.",
+  "keywords": [
+    "postal",
+    "address",
+    "dynamic",
+    "field",
+    "form"
+  ],
+  "files": [
+    "dist/jquery.addressfield.js",
+    "dist/jquery.addressfield.min.js",
+    "docs/**/*"
+  ],
+  "license": "MIT",
+  "main": "dist/jquery.addressfield.js",
   "devDependencies": {
-    "grunt-contrib-jshint": "~0.6.0",
-    "grunt-qunit-istanbul": "~0.4.5",
+    "grunt": "~0.4.4",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-clean": "~0.4.0",
     "grunt-contrib-concat": "~0.3.0",
+    "grunt-contrib-connect": "~0.7.1",
     "grunt-contrib-copy": "~0.6.0",
+    "grunt-contrib-jshint": "~0.6.0",
     "grunt-contrib-uglify": "~0.2.0",
     "grunt-contrib-watch": "~0.4.0",
-    "grunt-contrib-clean": "~0.4.0",
-    "grunt": "~0.4.4",
-    "grunt-contrib-connect": "~0.7.1"
+    "grunt-qunit-istanbul": "~0.4.5"
```

Comment:
Any reason not to dump the `libs` directory completely and just have a `devDependency` on `qunitjs ~1.11.0`?